### PR TITLE
fix: sign up maestro test

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
- Fixes new onboarding sign up test. 
- This test is not 100% perfect as we skip the captcha, but not sure what we can do about it. We can potentially create a dummy insta or twitter account for testing 🤔

# How
Updated maestro sign up test flow

# Test Plan
Test everything passes on maestro

### CODE REVIEW

File: packages/app/components/onboarding/hcaptcha/challenge.tsx [hunk 1]
```diff
@@ -36,6 +36,12 @@ export const Challenge = () => {
   const captchaRef = useRef<ConfirmHcaptcha>(null);
 
   const showCaptcha = () => {
+    // we don't need captcha on e2e tests
+    if (process.env.E2E) {
```
Comment (intergalacticspacehighway) [inline packages/app/components/onboarding/hcaptcha/challenge.tsx:40]: we should avoid changing code for tests. But I couldn't find any better way to do this. We can end the test on captcha screen though or create a dummy twitter/insta accounts.

Comment (hirbod) [reply packages/app/components/onboarding/hcaptcha/challenge.tsx:40]: There is another way: hCaptcha has a "dummy" sitekey which always results to true, we could use that maybe?

Comment (hirbod) [reply packages/app/components/onboarding/hcaptcha/challenge.tsx:40]: But LGTM anyway

**Review by hirbod — APPROVED** [review]

File: packages/app/components/onboarding/hcaptcha/challenge.tsx (hunk 1 above)
Comment (intergalacticspacehighway) [reply packages/app/components/onboarding/hcaptcha/challenge.tsx:40]: oh wow. There is? sure we should use that!

**Review by alantoa — APPROVED** [review]
LGTM

**Pull request merged**